README.md

```diff
@@ -50,6 +50,11 @@ cd src
 make
 cd ..
 ```
+or with cmake:
+```
+cmake src -B lib -DCMAKE_BUILD_TYPE=Release
+cmake --build lib
```

Comment:
```suggestion
cmake --build lib --clean-first --parallel 8
```

Some optional flags I find quite useful. I normally like to use `--clean-first --parallel $(($(nproc) - 1))` but that might be a bit extreme for some people, so `8` or less is probably fine.